src/editor/actions/delete_action.cpp

```
@@ -30,13 +30,13 @@ void DeleteAction::Undo() {
     // need to merge the layout entity trees too
     auto parentLayoutEntity = std::static_pointer_cast<moth_ui::LayoutEntityGroup>(m_parentNode->GetLayoutEntity());
     auto layoutEntity = m_deletedNode->GetLayoutEntity();
-    auto const safeIndex = std::min(m_originalIndex, parentLayoutEntity->m_children.size());
+    int const safeIndex = std::min(m_originalIndex, static_cast<int>(parentLayoutEntity->m_children.size()));
```

Clang (14.0.6)

[error] variable 'safeIndex' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)